### PR TITLE
Friends setup

### PR DESCRIPTION
**Pull request by AkshayPall**

### CODE REVIEW

File: server/data/models/friend.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      type: Type.INTEGER,
+      allowNull: false,
+    },
+    action_user: {
+      type: Type.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    freezeTableName: true,
```
Comment (eric-zhu-uw) [inline server/data/models/friend.js]: don't freeze table name since we want it to be FRIENDS

File: server/data/models/friend.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Type from 'sequelize';
+import sequelize from '../sequelize';
+
+const Friend = sequelize.define(
+  'Friend',
+  {
+    user_1: {
+      primaryKey: true,
+      type: Type.STRING,
```
Comment (eric-zhu-uw) [inline server/data/models/friend.js]: username is of Type.STRING(20) so user_1 should be Type.STRING(20)

File: server/data/models/friend.js (hunk 1 above)
Comment (AkshayPall) [reply server/data/models/friend.js]: Sure

File: server/data/models/friend.js (hunk 2 above)
Comment (AkshayPall) [reply server/data/models/friend.js]: k

**Pull request merged**